src/components/CheckboxRadioSwitch/CheckboxRadioSwitch.vue

```diff
@@ -133,6 +154,9 @@ export default {
 			'checkbox-radio-switch--checked': isChecked,
 			'checkbox-radio-switch--disabled': disabled,
 			'checkbox-radio-switch--indeterminate': indeterminate,
+			'checkbox-radio-switch--button-variant': buttonVariant,
+			'checkbox-radio-switch--button-variant-v-grouped': buttonVariantGrouped === 'vertical',
+			'checkbox-radio-switch--button-variant-h-grouped': buttonVariantGrouped === 'horizontal',
```

Comment:
Minor nitpick:
```suggestion
			'checkbox-radio-switch--button-variant-v-grouped': buttonVariant && buttonVariantGrouped === 'vertical',
			'checkbox-radio-switch--button-variant-h-grouped': buttonVariant && buttonVariantGrouped === 'horizontal',
```
because at the moment you can have `buttonVariantGrouped="horizontal"` while `:buttonVariant="false"` which gives `margin-left: -2px` to an active checkbox.

Reply:
:+1: